app/assets/stylesheets/content/editor/_markdown.sass

```diff
@@ -55,3 +55,9 @@ div.ck-editor__preview
     > table
       height: 100%
       width: 100%
+
+    // Ensure we break apart words in table cells that
+    // are restricted in width (OP#33524)
+    td[style*=";width:"],
+    td[style^="width:"]
```

Comment:
Although I do not like it, I guess we have to add the same rules for `th` elements, too. Otherwise width restrictions made in the header will still be applied.

Reply:
Good point, I did add that